include/megaapi.h

```diff
@@ -5250,14 +5250,7 @@ class MegaApi
         /**
          * @brief Add a new contact to the MEGA account
          *
-         * The associated request type with this request is MegaRequest::TYPE_ADD_CONTACT
-         * Valid data in the MegaRequest object received on callbacks:
-         * - MegaRequest::getEmail - Returns the email of the contact
-         *
-         * @param email Email of the new contact
-         * @param listener MegaRequestListener to track this request
-         *
-         * @deprecated: This way to add contacts will be removed in future updates. Please use MegaApi::inviteContact.
+         * @obsolete: This way to add contacts is obsolete. Please use MegaApi::inviteContact.
          */
         void addContact(const char* email, MegaRequestListener* listener = NULL);
```

Comment:
It seems that this function has been removed in `megaapi.cpp`, so it should be removed here too.